README.rst

```diff
@@ -131,6 +131,11 @@ Partition options:
 :bootable:		Boolean specifying whether to set the bootable flag.
 :in-partition-table:	Boolean specifying whether to include this partition in
 			the partition table. Defaults to true.
+:forced-primary:	Force this partition to be a primary partition in the
+			MBR partition table, useful when the extended partition should be
+			followed by primary partitions. If there are more partitions
+			defined after the first forced-primary, they must be also defined
+			as forced-primary. Defaults to false.
```

Comment:
Didn't you meant to say the opposite here? 

```suggestion
:forced-primary:	Force this partition to be a primary partition in the
			MBR partition table, useful when primary partitions should follow
			the extended partition. If there are more partitions defined
			after the first forced-primary, they must be also defined as
			forced-primary. Defaults to false.
```

Reply:
Right. Passive tense kinda confused me, but it is nicer as then the words are in the right order as well. So fine by me as is :+1: